### ISSUE TITLE
Error Copying Token from response body

### ISSUE THREAD

**Issue by Hexacker** [rangav/thunder-client-support#1223]
**Describe the bug**

I am copying a token from a login response body to an environment variable, and the result of that leads to a different result. It always copies the (") at the beginning of the token and somehow when I delete it, I get an error message of "invalid signature".

Without changing anything, it works perfectly on Postman.

**To Reproduce**

**Expected behavior**

**Platform:** 
 - OS: MacOS 13.4.1
 - vscode version: 1.80.0
 - extension version: v2.9.0

**Your Team Size Using TC:** 

1

**Comment by rangav** [comment]
Hi @Hexacker

Can you please share a sample response that is useful to re-produce the issue?

You can use [removeQuotes](https://github.com/rangav/thunder-client-support/blob/master/docs/filters.md#removequotes) filter to remove any quotes

![image](https://github.com/rangav/thunder-client-support/assets/8637550/56cb5d34-c033-45ac-9105-f77b2e6f48c9)

**Comment by Hexacker** [comment]
First, when generating the auth token, I got a normal response.
<img width="2680" alt="SCR-20230713-lewu" src="https://github.com/rangav/thunder-client-support/assets/1835592/fd3cd1cd-57ac-413f-8113-0d0f5a230390">

Copying the auth token to be an env var, the issue appears
<img width="2031" alt="SCR-20230713-lffx" src="https://github.com/rangav/thunder-client-support/assets/1835592/e33d4591-4b68-45fe-a745-176180776012">

The (") shows only at the beginning of it

Even after deleting the (") and the space, I get the error below(since it's not what I need exactly)
<img width="992" alt="SCR-20230713-lfle" src="https://github.com/rangav/thunder-client-support/assets/1835592/a518fc55-9165-404a-acb2-ba1a572aac81">

If I do the same thing using Postman, it works just fine. Also when copying the token from Postman and pasting it into Thunder Client environment variable.

**Comment by Hexacker** [comment]
I just discovered that anything I copy from the response body is copied with a space followed by a double quote (` "`)

**Comment by rangav** [comment]
How did you copy the token to Environment 

- manually copying response text?
- Or by using Tests tab - Set Env Variable?

**Comment by rangav** [comment]
Could be a VS Code bug, 

please restart VS Code and try again

**Comment by Hexacker** [comment]
The same thing, after several restarts of VSC

**Comment by rangav** [comment]
Copy some partial text in the response and paste in env?

**Comment by Hexacker** [comment]
The same issue remains

**Comment by rangav** [comment]
Dont know why its happening to you. 

May be reboot mac and try.

can you share video screen recordings of the issue? Could be useful to investigate further

**Comment by Hexacker** [comment]
Sure, I will record a video about all the weird issues

**Comment by Hexacker** [comment]
You can notice that there is a weird issue while writing on the request body.
BTW, all of the other extension was disabled.
https://github.com/rangav/thunder-client-support/assets/1835592/4de05c33-edc5-4a3c-9ba7-f77369f1e448

**Comment by rangav** [comment]
I think the copy paste functionality is provided by vscode, so could be VS code bug.

alternatively you can automate save response token to env var using the `Tests` tab -> Select `Set Env Variable`

**Comment by Hexacker** [comment]
When I copy from anywhere else in VSC, I don't have any issues at all.

Have you noticed the issue when writing or deleting characters in the request body?

**Comment by rangav** [comment]
I haven't noticed anything strange in the body.

Did you had this issue before?

**Comment by Hexacker** [comment]
Notice that when I was writing, it's writing in a very strange way, also when I delete characters, it's deleting in a very strange way.

I'm not sure if this is an old issue since I wasn't a hardcore user 😅

**Comment by rangav** [comment]
Did you change any font settings, usually happens when font settings are changed

**Comment by Hexacker** [comment]
Yes, I use a custom font in VSC, but that doesn't affect VSC or other extensions in any way

**Comment by rangav** [comment]
The editors in Thunder Client will use the VS Code font, some fonts are not properly supported in TC editors.

Try using the default Font or any other font and test it.

**Comment by Hexacker** [comment]
That's really weird, it works perfectly fine with the default font of VSC(which I can't use 😅).

Do you recommend any good font that's compatible with Thunder Client?
